src/components/SprintSection-1/SprintSection-1.module.scss

```diff
@@ -0,0 +1,178 @@
+@import '../../styles/color.scss';
+
+:export {
+  cyanDark: $cyanDark;
+  cyanLight: $cyanLight;
+}
+
+.body_section {
+  display: flex;
+  padding: 100px 528px;
+  width: 1920px;
+  height: 640px;
+
+  /* Primaty/Grey Light */
+  background: #F6F5F4;
+}
+
+.sprint_section {
+  display: flex;
+  align-items: center;
+  padding: 20px 40px;
+  gap: 30px;
```

Comment:
https://github.com/taodemy/taotalk/blob/46b32615e8c8326691690996c98c8f4b4c54b755/src/components/SprintSection-1/SprintSection-1.module.scss#L17-L21

Modified as suggests. 

Thanks for your careful checking! Because this error in gap setting lets the picture size shrink. And I debugged it for a long time. And I even asked Tao Zong for setting max/min-width for some responsive design...